rocky/rocky/templates/oois/ooi_list.html

```diff
@@ -37,13 +37,64 @@ <h1>
                     {% blocktranslate with length=ooi_list|length total=total_oois %}Showing {{ length }} of {{ total }} objects{% endblocktranslate %}
                 </p>
                 <form method="post"
-                      class="inline layout-wide checkboxes_required"
-                      action="#bulk_clearance_level_submit">
+                      id="ooi_list_form"
+                      class="inline layout-wide checkboxes_required">
                     {% csrf_token %}
                     <div class="horizontal-scroll">
-                        {% include "partials/form/form_errors.html" with form=select_oois_form %}
+                        <table class="nowrap">
+                            <caption class="visually-hidden">{% translate "Objects" %}</caption>
```

Comment:
You could consider adding a short visually-hidden explanation about the buttons in the headers here. Something like "column headers with buttons are sortable".

See:
- the [ARIA APG sortable table example](https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/sortable-table/) example
- the [Manon docs sortable table example](https://minvws.github.io/nl-rdo-manon/components/table-sortable) (Dutch)